assets/app/data/crowbar/services/crowbar.factory.js

```diff
@@ -0,0 +1,84 @@
+(function() {
+
+    angular
+        .module('suseData.crowbar')
+        .factory('crowbarFactory', crowbarFactory);
+
+    crowbarFactory.$inject = ['$q', '$http', 'COMMON_API_V2_HEADERS'];
+    /* @ngInject */
+    function crowbarFactory($q, $http, COMMON_API_V2_HEADERS) {
+        var factory = {
+            getEntity: getEntity,
+            getRepositoriesChecks: getRepositoriesChecks,
+            upgrade: upgrade,
+            getUpgradeStatus: getUpgradeStatus
+        };
+
+        return factory;
+
+        /**
+         * Get Crowbar Entity
+         * 
+         * @return {Promise}
+         */
+        function getEntity() {
+
+            var requestOptions = {
+                method: 'GET',
+                url: '/api/crowbar',
+                headers: COMMON_API_V2_HEADERS
```

Comment:
Something like https://github.com/mgonto/restangular#how-to-create-a-restangular-service-with-a-different-configuration-from-the-global-one